parsl/executors/high_throughput/executor.py

```diff
@@ -204,6 +207,10 @@ def initialize_scaling(self):
         debug_opts = "--debug" if self.worker_debug else ""
         max_workers = "" if self.max_workers == float('inf') else "--max_workers={}".format(self.max_workers)
 
+        worker_logdir = "{}/{}".format(self.run_dir, self.label)
+        if self.worker_logdir_root is not None:
+            worker_logdir = "{}/{}".format(self.worker_logdir_root, self.label)
```

Comment:
we need to make sure this directory exists somewhere

Reply:
This should be handled in process_worker_pool.py, no?

Reply:
If so, I'd rather leave that as an issue to follow up, I don't want to muck around in that too much.